tests/raft_scenarios/5879_deprecated

```diff
@@ -0,0 +1,65 @@
+nodes,0,1
+connect,0,1
+
+# Node 0 starts first, and begins sending messages first
+periodic_one,0,110
+dispatch_all
+
+periodic_all,30
+dispatch_all
+
+state_all
+assert_is_primary,0
+assert_is_backup,1
+
+## ... with the same logs upto index 7.
+replicate,1,txAtIdx1
+replicate,1,txAtIdx2
+replicate,1,txAtIdx3
+replicate,1,txAtIdx4
+replicate,1,txAtIdx5
+replicate,1,txAtIdx6
+emit_signature,1
+periodic_all,30
+dispatch_all
+
+assert_commit_idx,0,7
+assert_commit_idx,1,0
+
+periodic_all,30
+dispatch_all
+assert_state_sync
+
+assert_commit_idx,0,7
+assert_commit_idx,1,7
+
+## At index 8, the leader n2 reconfigures to remove n1.
```

Comment:
I don't think we need reconfiguration to show this, a scenario with just two pending signatures in the log on a stable config is enough to get the "wrong" commit (i.e. the first signature, not the second one).

Reply:
Yes, reconfiguration is not needed to show that the spec does not advance the commit index enough. This scenario was to test whether this PR always advances the commit index as much as it can